src/argilla/client/feedback/schemas/records.py

```diff
@@ -52,6 +53,12 @@ class ValueSchema(BaseModel):
     value: Union[StrictStr, StrictInt, List[str], List[RankingValueSchema]]
 
 
+class ResponseStatus(str, Enum):
+    draft = "draft"
+    submitted = "submitted"
+    discarded = "discarded"
```

Comment:
couldn't we reuse the one from the SDK that you have added below?

Reply:
You mean to just add the `missing` status for the filtering and inheriting from this one?